crates/client/db/src/mapping_db.rs

```diff
@@ -45,30 +46,31 @@ impl MappingDb {
     /// Under some circumstances it can return multiples blocks hashes, meaning that the result has
     /// to be checked against the actual blockchain state in order to find the good one.
     pub fn block_hash(&self, starknet_block_hash: StarkHash) -> Result<Option<Vec<DHashT>>, DbError> {
-        match self.db.get(crate::columns::BLOCK_MAPPING, &starknet_block_hash.encode()) {
+        let block_mapping_col = self.db.get_column(Column::BlockMapping);
+
+        match self.db.get_cf(&block_mapping_col, starknet_block_hash.encode())? {
             Some(raw) => Ok(Some(Vec::<DHashT>::decode(&mut &raw[..])?)),
             None => Ok(None),
         }
     }
 
     /// Register that a Substrate block has been seen, without it containing a Starknet one
     pub fn write_none(&self, block_hash: DHashT) -> Result<(), DbError> {
```

Comment:
Do we know what this is used for? The name kinda screams 'delete me' from my point of view haha. Maybe in a later pr, just wondering.

Reply:
i really have no clue :)